jenkins/jobs/image-mint.yaml

```diff
@@ -40,7 +41,7 @@
         suite=bionic
         if [ "${release}" = "ulyana" ] || [ "${release}" = "ulyssa" ] || [ "${release}" = "uma" ] || [ "${release}" = "una" ]; then
             suite=focal
-        elif [ "${release}" = "vanessa" ]; then
+        elif [ "${release}" = "vanessa" ] || [ "${release}" = "vera" ]; then
```

Comment:
Can this be an else?

Reply:
No. The default is `bionic` for `tara`, `tessa`, and `tina`. If this were an `else`, those `t`-releases would also use `jammy`.

Reply:
Sounds like its worth re-organising so that the older exceptions are explicitly stated, and the newer ones just default to jammy.

Reply:
I don't think it's worth re-organizing now. The `t`-releases will go away in April 2023, then we can switch to an `else`.